src/styles/blocks/card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 1px;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  &__product-image {
+    width: 160px;
+    margin: 0 3px;
+    height: 134px;
+  }
+
+  &__product-name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $primary-color;
+    text-align: left;
+    margin-top: 40px;
+  }
+
+  &__product-code {
+    color: $secondary-color;
+    width: 100%;
+    margin-top: 4px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    width: 100%;
+    margin-top: 16px;
+
+    &-amount {
+      color: $primary-color;
+      margin: 0;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    margin-top: 24px;
+  }
+
+  &__label-price {
+    font-size: 12px;
+    line-height: 18px;
+    color: $secondary-color;
+  }
+
+  &__actual-price {
+    font-size: 16px;
+    line-height: 18px;
+    font-weight: 700;
+    color: $primary-color;
+  }
+
+  &__buy-button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-transform: uppercase;
+    text-decoration: none;
+    font-size: 14px;
+    line-height: 16px;
+    font-weight: 700;
+    margin-top: 16px;
+    color: $secondary-button-buy-color;
+    background-color: $primary-button-buy-color;
+    height: 40px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. Use padding instead